packages/components/search/src/search.css

```diff
@@ -0,0 +1,40 @@
+/* General........................................................ */
+
+.search-container {
+	background-color: var(--ts-color-white);
+	border: var(--ts-border);
+	border-radius: var(--ts-unit-double);
+	display: flex;
+	height: var(--ts-unit-double);
+	width: 100%;
+
+	&.focused {
+		border: var(--ts-border-focus);
+	}
+}
+
+input {
+	border: 0;
+	flex: 1;
+	font-size: var(--ts-font-size-default);
+	line-height: var(--ts-unit);
+
+	/* hide a default clear input button in IE */
+	&::-ms-clear {
+		display: none;
+	}
+}
+
+ts-icon {
+	flex: 0 1 0%; /* "%" as a workaround for IE 10-11 bug https://github.com/philipwalton/flexbugs#flexbug-4 */
+	--ts-icon-color: var(--ts-color-slate-lightest);
+}
+
+.search-icon {
+	margin: 7px;
```

Comment:
What's the reason for this number?

Reply:
Yeah, it actually a result of some non-trivial calculations. I replaced it with calc expression and added a comment with some details.